code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -48,6 +49,11 @@
 	var/mob/living/simple_animal/bot/active_bot
 	var/list/botlist = list()
 
+/obj/item/weapon/cartridge/New(var/obj/item/device/pda/pda)

```

Comment:
[Mainly to be easier to control when you want these things to happen(map loaded or later)](https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#prefer-initialize-over-new-for-atoms)